lib/utils/store.dart

```diff
@@ -21,18 +21,32 @@ class User {
 class Store extends GetxController {
   IOWebSocketChannel channel;
   User user = User();
+  bool isConnectionError = false;
 
   static Store get to => Get.find();
 
-  Future<void> createConnection(String ipAddr, String username) async {
-    var url = "http://$ipAddr:41431/register";
-    var body = {
-      "username": username
-    };
-
-    await http.post(url, body: body);
-
-    this.user.connect(username);
-    this.channel = IOWebSocketChannel.connect("ws://$ipAddr:41431/ws/$username");
+  Future<http.Response> createConnection(String ipAddr, String username) async {
+    var url = "https://$ipAddr:41431/register";
```

Comment:
It hasn't reached a stable version yet so let's keep it `http` for now since we are trying to connect with a local server that can be running on the user's computer. It is a breaking change so will take a while and it **may** allow making connections with servers on the same network, if not then we'll have to find another workaround.

Reply:
Yes got it.
I have made changes, could you please review it?